python_modules/dagster-graphql/dagster_graphql/schema/backfill.py

```diff
@@ -107,12 +107,25 @@ class Meta:
         name = "ResumeBackfillSuccess"
 
 
+class GrapheneRetryBackfillSuccess(graphene.ObjectType):
+    backfill_id = graphene.NonNull(graphene.String)
+
+    class Meta:
+        name = "RetryBackfillSuccess"
+
+
 class GrapheneResumeBackfillResult(graphene.Union):
     class Meta:
         types = (GrapheneResumeBackfillSuccess, GrapheneUnauthorizedError, GraphenePythonError)
         name = "ResumeBackfillResult"
 
 
+class GrapheneRetryBackfillResult(graphene.Union):
```

Comment:
for runs we have different Graphene result types for different actions (launch, reexecute), so i used that pattern here as well. it seems simpler to have a single `GrapheneLaunchBackfillResult` but i'm not sure if there's a reason for having the types split out like this